airgun/entities/base.py

```diff
@@ -6,3 +9,32 @@ def __init__(self, browser):
         self.browser = browser
         self.session = browser.extra_objects['session']
         self.navigate_to = self.session.navigator.navigate
+
+    def create_bookmark(self, values, search_query=None):
+        """Create a bookmark.
+
+        :param dict values: dictionary with keys 'name', 'query', 'public'
+        :param str optional search_query: if something different from `query`
+            should be typed into searchbox
+        """
+        # not using separate navigator step here not to have to register
+        # navigate step for every single entity
+        view = self.navigate_to(self, 'All')
+        if not search_query:
```

Comment:
doing
```
        if not search_query:
            search_query = values.get('query')
        view.search(search_query)
```
you completely dismiss the whole point of `search_query` and `query` parameters difference.

You whether should permit next scenario:
1) I fill search query and then query is filled automatically
2) I fill nothing for search query and fill query manually

or not:
1) I can fill only search query. Then it doesn't make sense to have a parameter and we should `query` directly into `search_query`

Reply:
Both use cases will work, although for second one searchbox will be filled anyways which is slight redundancy.
To be clear, are you suggesting
```python
if search_query:
    view.searchbox.search_field.fill(search_query)
```
instead?

Reply:
Nope, both scenario can't work as they mutually exclusive, but I thought and read more your code. And yes, exacly:
```
if search_query:
    view.searchbox.search_field.fill(search_query)
```